File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Settings/ISettingData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Settings
{
    public interface ISettingData :  CoreObject.IObjectID, CoreObject.IObjectTitle, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState

    {
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Settings/ISettingData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Settings;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ISettingData extends CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
